src/inmanta/graphql/schema.py

```diff
@@ -101,14 +61,60 @@ def add_filter_and_sort(
     return stmt
 
 
-def initialize_schema() -> strawberry.Schema:
+def get_schema(metadata: GraphQLMetadata) -> strawberry.Schema:
     """
     Initializes the Strawberry GraphQL schema.
     It is initiated in a function instead of being declared at the module level, because we have to do this
     after the SQLAlchemy engine is initialized.
     """
+
+    mapper: StrawberrySQLAlchemyMapper[typing.Any] = StrawberrySQLAlchemyMapper()
```

Comment:
Is this an expensive operation? Because this would be executed for each test case, if I understand correctly.

Reply:
That is correct, I extracted it from the method. Doesn't seem to be very expensive